dependencies/kotlin-build-gradle-plugin/build.gradle.kts

```diff
@@ -29,10 +29,15 @@ java {
 
 sourceSets {
     main {
-        java.setSrcDirs(listOf("src"))
+        /*TODO: move version to build-plugin*/
```

Comment:
Lets fix it before merge

Reply:
no, we've agreed to avoid movements before merge

Reply:
KTI-479 buildSrc and kotlin-build-gradle-plugin re-organization after Merge